bee-ternary/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "bee-ternary"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
```

Comment:
Why are we using the German term? Why not "IOTA Foundation"?

Reply:
This was in here when I started working on the crate. I'll change it.

Reply:
wait